openssl/build.sh

```diff
@@ -35,6 +35,8 @@ make -C "$PREFIX_OPENSSL_BUILD" install_sw
 cp -a "$PREFIX_OPENSSL_INSTALL/include/openssl" "$PREFIX_H"
 cp -a "$PREFIX_OPENSSL_INSTALL/lib/libcrypto.a" "$PREFIX_A"
 cp -a "$PREFIX_OPENSSL_INSTALL/lib/libssl.a"  "$PREFIX_A"
+cp -a "$PREFIX_OPENSSL_INSTALL/lib/pkgconfig" "$PREFIX_A"
+sed -i "s/openssl\/install$/lib\/pkgconfg/" "$PREFIX_A/pkgconfig/"*
```

Comment:
AFAIK, this will not work on Darwin (macOS) - `sed` does not have `-i`. See `lighttpd` build.sh where `ex` is being used instead.

Personally, I don't care, LGTM :).